packages/expo-dev-launcher/bundle/components/EmptyBranchesMessage.tsx

```diff
@@ -0,0 +1,86 @@
+import { Spacer, View, Text, Button, Heading } from 'expo-dev-client-components';
+import * as React from 'react';
+
+import { useBuildInfo } from '../providers/BuildInfoProvider';
+import { Branch } from '../queries/useBranchesForApp';
+
+type EmptyBranchesMessageProps = {
+  branches: Branch[];
+  incompatibleBranches: Branch[];
+  onLearnMorePress: () => void;
+};
+
+export function EmptyBranchesMessage({
+  branches = [],
+  incompatibleBranches = [],
+  onLearnMorePress,
+}: EmptyBranchesMessageProps) {
+  const { runtimeVersion } = useBuildInfo();
+
+  // no compatible branches
+  if (branches.length === 0 && incompatibleBranches.length > 0) {
+    const recentBranchWithUpdates = incompatibleBranches.find(
+      (branch) => branch.updates.length > 0
+    );
+    const latestRuntimeVersion = recentBranchWithUpdates?.updates?.[0].runtimeVersion;
+
+    return (
+      <View padding="medium" rounded="large" bg="default">
+        <View>
+          <Heading>The runtime version of this app has no compatible branches.</Heading>
```

Comment:
```suggestion
          <Heading>There are no branches compatible with this development build.</Heading>
```
i think simpler is better here, in terms of language